lib/hocon/impl/simple_config_list.rb

```diff
@@ -21,6 +24,8 @@ def initialize(origin, value, status = ResolveStatus.from_values(value))
     end
   end
 
+  def_delegators :@value, :[], :include?, :empty?, :size, :index, :rindex, :each, :map
+
```

Comment:
I'm fine either way. I can definitely see how it would make it harder to maintain in the future, since it's confusing to look at